cluster/cluster.go

```diff
@@ -97,7 +101,7 @@ func Start(numInstances int) error {
 // Restart the cluster
 func Restart(ctx context.Context) error {
 	for i := 0; i < len(daemons); i++ {
-		daemons[i].Close()
+		daemons[i].Close(context.Background())
```

Comment:
For both Restart() and Close() (and the others, it seems) - do these ctx instances need timeouts?